integration-tests/src/bin/write.rs

```diff
@@ -0,0 +1,11 @@
+// SPDX-License-Identifier: Apache-2.0
+
+use core::ffi::c_void;
+use libc::write;
+
+fn main() {
+    let buf = "hello world\n".as_ptr() as *const c_void;
+    unsafe {
+        write(1, buf, 12);
```

Comment:
```suggestion
        write(libc::STDOUT_FILENO, buf.as_ptr() as *const _, buf.len().into());
```

Reply:
I was looking for something like `libc::STDOUT_FILENO`! Thank you @npmccallum .